civis/ml/_helper.py

```diff
@@ -0,0 +1,40 @@
+from civis import APIClient
+from civis.ml._model import _PRED_TEMPLATES
+
+
+def list_models(job_type=None, client=None, **kwargs):
```

Comment:
What would you think of defaulting to `"train"`? When I think of a "model", usually it's the training job rather than the scoring job.

Reply:
Yeah, I went back and forth between defaulting to "train" vs None. Happy to change it.